addon/components/rdf-input-fields/input/edit.js

```diff
@@ -1,13 +1,52 @@
 import { action } from '@ember/object';
 import { guidFor } from '@ember/object/internals';
 import SimpleInputFieldComponent from '../simple-value-input-field';
+import { FORM, SHACL } from '@lblod/submission-form-helpers';
+import { next } from '@ember/runloop';
 
 export default class FormInputFieldsInputEditComponent extends SimpleInputFieldComponent {
```

Comment:
So I see allot of shared code between the two input-fields. Normally we place these kind off code in the abstract class `InputFieldComponent`.